tests/test_jjm_write_task_result.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+import mock
+import os
+
+import os.path
+
+from thor.maestro.jenkins import JenkinsJobManager
+from thor.dao.task_dao import read_task, get_task_num
+from thor.dao.clear_tables_reseed import reseed
+
+from thor.dao.release_dao import release_id_lookup_class
+from thor.dao.models import Task
+
+
+def returnSuccess(self, input1, input2):
```

Comment:
For these hardcoded outputs that are not as lengthy as the expected JSON  responses but are also for our assertions, maybe you could move these functions to `tests/conftest.py` as a pytest `fixture` (https://docs.pytest.org/en/latest/explanation/fixtures.html).

e.g.,

```
class Mocker(object):
    def mock_functions(self):
        self.patcher = mock.patch("thor.maestro.jenkins.JenkinsJobManager.check_result_of_job", return_value="success")

@pytest.fixture(scope="function")
def jenkins_job_manager_instance():
    mocker = Mocker()
    mocker.mock_functions()
    
```

and then, in your test function you pass the fixture as an argument:
e.g.,

```
@mock.patch.object(release_id_lookup_class, "release_id_lookup", returnThree)
def test_write_no_prior(jenkins_job_manager_instance):
    #  jenkins_job_manager_instance.check_result_of_job should be patched here... 
```

Something around these lines..
I'm trying to steal the success from our Fence service here:
https://github.com/uc-cdis/fence/search?q=mocker

But this is a way more complex project.
You don't need to do this, but it would be pretty awesome and make our project more elegant.

Reply:
Tried copying the code and was unable to get it to work, I'll leave it in there commented out and come back to it in a later PR.